license-bill-of-materials.go

```diff
@@ -415,50 +417,54 @@ func listLicenses(gopath string, pkgs []string) ([]License, error) {
 	// subpackages like bleve.
 	matched := map[string]MatchResult{}
 
-	licenses := []License{}
+	gpackages := []GoPackage{}
```

Comment:
We using camel case to define variable names. This should be gPackages or just packages would suffice